.eslintignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+*.test.js
```

Comment:
Why test files are ignored for Eslint. We should keep linter enabled for all JS files

Reply:
Ou, it is my bad.